Extension du TwitchTracker Scraper

Hicham77500 opened the pull request:
📋 Résumé
- Extension du projet de scraping TwitchTracker pour ajouter 2 nouveaux scrapers tout en gardant le projet propre et fidèle à l'original.

🚀 Fonctionnalités ajoutées
- Scraper Top 500 mondial 

- Structure identique au scraper FR
- Scrape les 500 streamers les plus populaires au monde

Extraction de bio, jeux joués, lives récents, stats mensuelles
- Fonctionne avec les données du Top 500 mondial
- Limité à 50 profils pour éviter les timeouts
📁 Fichiers ajoutés
scraper/
├── viewership_world.py     # NOUVEAU - Top 500 mondial
└── individual_profiles.py  # NOUVEAU - Profils individuels
📝 Fichiers modifiés

main.py - Menu unifié avec 4 options :

France uniquement
- Top 500 mondial uniquement
- Tout (France + Mondial)
- Profils individuels (Top 50)


insert_data.py - Ajout de 2 nouvelles fonctions :
- insert_viewership_world_data()
- insert_individual_profiles_data()

✅ Scraping du Top 500 mondial
✅ Scraping des profils individuels des streamers

Hicham77500 committed:
Scraper Monde + Scraper Profiles

Hicham77500 committed:
Mise a jour readme

Calamia92 commented on __pycache__/main.cpython-313.pyc:
Tu as push les fichiers _pycache, il ne sont pas nécessaires et ne doivent pas être push. Tu dois les mettre dans le .gitignore

Calamia92 commented on main.py:
Le menu interactif basé sur input() reflète une mauvaise approche dans ce contexte : le projet doit maintenir des données de rankings/statistiques à jour en permanence, et ce type d’interaction humaine bloque toute automatisation. Il faudrait inverser la logique et considérer que tous les scrapers doivent s’exécuter automatiquement, sans intervention manuelle, afin de garantir des données fraîches et fiables.

Calamia92 commented on scraper/__pycache__/individual_profiles.cpython-313.pyc:
Tu as push les fichiers _pycache, il ne sont pas nécessaires et ne doivent pas être push. Tu dois les mettre dans le .gitignore

Calamia92 commented on scraper/__pycache__/viewership_world.cpython-313.pyc:
Tu as push les fichiers _pycache, il ne sont pas nécessaires et ne doivent pas être push. Tu dois les mettre dans le .gitignore

Calamia92 commented on scraper/viewership_fr.py:
Pourquoi avoir changer mon script ? J'attends des arguments.

Calamia92 committed:
Remove Python cache files and update .gitignore

Deleted unnecessary .pyc files from __pycache__ directories and added them to .gitignore to prevent future commits of compiled Python files.
